travvy/src/store/store.js

```diff
@@ -26,6 +32,21 @@ export default new Vuex.Store({
         //when a user logs in, set the variable to their email
         setUserEmail(state,userEmail){
             state.userEmail = userEmail
+        },
+        setAge(state,age){
+            state.age = age
+        },
+        setGender(state,gender){
+            state.gender = gender
+        },
+        setInstagram(state,instagram){
+            state.instagram = instagram
+        },
+        setName(state,name){
+            state.name = name
+        },
+        setActivity(state,activity){
```

Comment:
This can be used later within our search algorithm since it is being kept in the 'store'